frontend/src/App.tsx

```diff
@@ -60,50 +60,54 @@ function MobileAppToolbar() {
   }
 
   return (
-    <Toolbar>
-      <IconButton onClick={() => setOpen(true)}>
-        <MenuIcon />
-      </IconButton>
-      <Drawer variant="persistent" anchor="left" open={open}>
-        <Button onClick={() => setOpen(false)}>
-          <ChevronLeftIcon />
-        </Button>
-        <List>
-          {Object.keys(PAGE_URL_TO_NAMES).map((pageUrl, index) => (
-            <ListItemLink href={pageUrl} key={index}>
-              <ListItemText primary={PAGE_URL_TO_NAMES[pageUrl]} />
-            </ListItemLink>
-          ))}
-        </List>
-      </Drawer>
-    </Toolbar>
+      <Toolbar>
+        <IconButton onClick={() => setOpen(true)}>
+          <MenuIcon />
+        </IconButton>
+        <Drawer variant="persistent" anchor="left" open={open}>
+          <Button aria-label="Hide site navigation button"
+                  onClick={() => setOpen(false)}>
+            <ChevronLeftIcon />
+          </Button>
+          <List>
+            {Object.keys(PAGE_URL_TO_NAMES).map((pageUrl, index) => (
+                <ListItemLink href={pageUrl} key={index}
+                              aria-label={PAGE_URL_TO_NAMES[pageUrl]}>
```

Comment:
I do not think you need an aria-label here, as the screen reader should announce the text value instead.